src/scancode/api.py

```diff
@@ -179,17 +181,23 @@ def get_licenses(location, min_score=0,
     match = None
     for match in matches:
         qspans.append(match.qspan)
-
-        detected_expressions.append(match.rule.license_expression)
-
-        detected_licenses.extend(
+        
+        matched_license=[]
+        matched_license.extend(
             _licenses_data_from_match(
                 match=match,
                 include_text=include_text,
                 license_text_diagnostics=license_text_diagnostics,
                 license_url_template=license_url_template)
         )
-
+        
+        if "unknown" in match.rule.license_expression:  #TODO: use is_unknown flag instead
```

Comment:
re `#TODO: use is_unknown flag instead` when would this happen?

Reply:
@pombredanne after https://github.com/nexB/scancode-toolkit/pull/2548 is approved and merged, then this could be rebased/merged and the `is_unknown` flags introduced there can be used.